src/createFix.ts

```diff
@@ -0,0 +1,98 @@
+import * as vscode from 'vscode';

```

Comment:
I don't think this file can work as a generic framework to add different autofixes to the `quickfixes.yaml` config file.
It seems very add-hoc to some particular cases.
Manipulating users workspace yaml files is a sensitive procedure that should easily tested and debuged.

- There should be a generic set of functions to manipulate yaml files by `JsonPath`: add, update, delete, append...
- Those function should be as much as posible agnostic of vscode APIs because vscode apis can not be easily unit-tested.
- Those function should be unit-tested. This was an important task: to investigate different unit testing libs for javascript (like mocha and jest), try them and then request for guidance to community members to decide about which one to use.